### PR TITLE
Blindij/gh actions

### LINKED ISSUE #54: GitHub Actions
I think we could simplify the lesson by demonstrating automated testing with GitHub actions:
- no new account to create
- nothing to enable and not a different tab to switch to
- better support for testing on macOS and Windows
- all you need to do is add a file, that's it
- we could point learners to the many examples for other languages

I can adjust lesson but don't want to change it now under @blindij feet.

But I think the minimum we need to do and this is what I would ask @blindij to do in the upcoming workshops is to at least mention it.

Otherwise we can look outdated. I still like and will use Travis but for many situations GH Actions is just simpler.

### PR DESCRIPTION
**Pull request by blindij**
- replace Travis with GitHub actions
- replace "type along" with "Exercise"
; closes #54

### CODE REVIEW

**Comment by bast** [comment]
I am looking now ...

File: _episodes/03-pytest.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+> E         +5
+> 
+> example.py:6: AssertionError
+> ========================================================= 1 failed in 0.05 seconds ==============
+> ```
+{: .challenge}
+
+---
+
+## Note to Fortran developers
```
Comment (bast) [inline _episodes/03-pytest.md:95]: I wrote this one but was always wondering whether this side-note is helpful or just distracting and confusing. Maybe we could completely remove it?

Comment (blindij) [reply _episodes/03-pytest.md:95]: Leave it for now, as a separation between the exercises. Someone will find it useful.

File: _episodes/05-gh-actions.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+### Step 11: Discussion
+
+We discuss together about our experiences with this exercise.
+
+---
+{: .challenge}
+
+## Where to go from here
+
+- This example was using Python but you can achieve the same automation for Fortran or C or C++
```
Comment (bast) [inline _episodes/05-gh-actions.md:192]: or R or another language.

File: _episodes/05-gh-actions.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+
+### Step 11: Discussion
+
+We discuss together about our experiences with this exercise.
+
+---
+{: .challenge}
+
+## Where to go from here
```
Comment (bast) [inline _episodes/05-gh-actions.md:190]: Also here we can link to the "marketplace" and encourage participants to find an interesting workflow which is close to their language/problem.

**Pull request merged**